nav2_behavior_tree/plugins/decorator/rate_controller.cpp

```diff
@@ -26,13 +26,14 @@ RateController::RateController(
 : BT::DecoratorNode(name, conf),
   first_time_(false)
 {
-  double hz = 1.0;
-  getInput("hz", hz);
-  period_ = 1.0 / hz;
 }
 
 BT::NodeStatus RateController::tick()
 {
+  double hz = 1.0;
```

Comment:
Why is this not wrapped by a check for initialization?

Reply:
done